.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -1,9 +1,6 @@
 name: Bug report
 description: Report a way in which WorldEdit is not working as intended
-title: ''
 labels: ['type:bug', 'status:pending']
-assignees: []
-issue_body: true
```

Comment:
Can we add the catchall "Anything else?" textarea to replace this, in both templates? I want to keep that.

Reply:
I did that here: https://github.com/EngineHub/WorldEdit/pull/1736/files#diff-637f7b97bba458badb691a1557c3d4648686292e948dbe3e8360564378b653efR55
Using `textarea` now makes use of the gh-typical WYSIWYG comment box.

Reply:
Hmm, yea. Not sure why we had `issue_body` then.